vars/ploigosWorkflowContainerScan.groovy

```diff
@@ -0,0 +1,474 @@
+#!/usr/bin/env groovy
```

Comment:
lets do `ploigosWorkflowExistingContainerImageScan` to make it clear its for scanning an already built container image.